### ISSUE TITLE
makeValid error

### ISSUE THREAD

**Issue by Trepon** [LAPKB/Pmetrics#23]
Hello,

When using the makeValid function, I get the following error:

```
$npde
<simpleError in decorr.chol(varsim): object 'ymat' not found>

attr(,"class")
[1] "PMvalid" "list" 
```

During the the npde computation, there was also an error message:

```
Computing the npde for subject  11 
Error in chol.default(x) : 
  the leading minor of order 3 is not positive definite
$simdata
$obs
```

I end up with "thisobs" and "thissim" objects in my environment, instead of what I suppose should be a PMvalid object.
I'm using the latest Pmetrics build (with the EVID=4 fix).

Here is the code I ran:
`makeValid(356,
  binCov = "agfr",
  limits = NA,
  nsim = 10,
  predint = 0,
  split = T
)`

Please find attached the data, model, and terminal output after running the makeValid function.

Thank you for your help

[model.txt](https://github.com/LAPKB/Pmetrics/files/5061695/model.txt)
[dataset3.zip](https://github.com/LAPKB/Pmetrics/files/5061721/dataset3.zip)
[makeValid error.txt](https://github.com/LAPKB/Pmetrics/files/5061686/makeValid.error.txt)

**Comment by mnneely** [comment]
Coincidentally, I just had the same bug myself and fixed it.  It is a problem in the npde package.  Your variance matrix was not positive-definite, which usually happens with sparse data or small populations.  I took the fix I made in Pmetrics and applied it to npde.  Just reinstall npde using this code in Rstudio:

```devtools::install_github("LAPKB/npde")```

That should work for you. Pmetrics will now ask if you want to fix it.

BTW, ```nsim=10``` generally isn't enough.  If you weren't just testing it, I would use 1000 (the default). Also, you don't need the ```predint``` argument.  This algorithm only simulates at times of observations. Likewise, ```split=T``` will be ignored, as the algorithm is testing the central tendency of the model, so doesn't split.
